migrations/m161204_150439_create_department_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles the creation of table `department`.
+ */
+class m161204_150439_create_department_table extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public function up()
+    {
+        $this->createTable('department', [
+            'id' => $this->primaryKey(),
+            'department_name' => $this->string(50)->notNull(),
```

Comment:
You don't need to add "department_" prefix, it's enough to call this field "name"